plugins/woocommerce-admin/client/marketplace/utils/functions.tsx

```diff
@@ -199,17 +203,114 @@ function installProduct( productKey: string ): Promise< void > {
 	} );
 }
 
-function connectProduct( productKey: string ): Promise< void > {
+function disconnectProduct( subscription: Subscription ): Promise< void > {
+	if ( subscription.active === false ) {
+		return Promise.resolve();
+	}
+	const url = '/wc/v3/marketplace/subscriptions/disconnect';
+	const data = new URLSearchParams();
+	data.append( 'product_key', subscription.product_key );
+	return apiFetch( {
+		path: url.toString(),
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/x-www-form-urlencoded',
+		},
+		body: data,
+	} );
+}
+
+function wpAjax(
+	action: string,
+	data: {
+		slug?: string;
+		plugin?: string;
```

Comment:
I am not seeing where these plugin and theme are being used. Could you point me to it ?

Reply:
It's part of WP https://github.com/WordPress/WordPress/blob/master/wp-admin/update.php#L23C6-L24C3

Reply:
AH got it ! Nevermind then!